src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -701,7 +726,8 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 SupportsLogPoints = true,
                 SupportsReadMemoryRequest = true,
                 SupportsModulesRequest = true,
-                AdditionalModuleColumns = additionalModuleColumns
+                AdditionalModuleColumns = additionalModuleColumns,
+                SupportsDisassembleRequest = true
```

Comment:
We need to double check if this is valid for `lldb-mi`.

Reply:
We should be ok:
https://github.com/lldb-tools/lldb-mi/blob/a9faab69fc30ac2bba0521bbcf9e4931071272cf/src/MICmdCmdData.h#L94